ingestion/examples/amundsen_data/service_type.json

```diff
@@ -0,0 +1,32 @@
+{
+    "hive": {
+        "service_name": "Hive",
+        "connection": {
+            "config": {
+                "hostPort": "http://nohost:6000",
+                "type": "Hive"
+            }
+        }
+    },
+    "delta": {
+        "service_name": "DeltaLake",
+        "connection": {
+            "config": {
+                "metastoreConnection": {
+                    "metastoreHostPort": "http://localhost:9083"
+                }
+            }
+        }
+    },
+    "dynamo": {
+        "service_name": "DynamoDB",
+        "connection": {
+            "config": {
+                "awsConfig": {
+                    "awsRegion": "aws_region"
+                },
+                "type": "DynamoDB"
+            }
+        }
+    }
+}
```

Comment:
there are many more data sources that supports (here is the [list](https://docs.open-metadata.org/connectors/metadata/amundsen#3-select-the-service-type)) we might need to add those here as well